docs/pages/push-notifications/what-you-need-to-know.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: What you need to know about notifications
+hideFromSearch: false
+description: Different kinds of notifications and notification behaviors you need to know before you get started.
+---
+
+Notifications have a large surface area and differences across platforms can make implementing notifications intimidating.
+
+Whether you are starting with notifications or have existing knowledge, this document aims to give you a quick start into the topic by explaining the different kinds of notifications and notification behaviors you should understand.
+
+Remember that Expo's support for notifications builds on top of native functionality for Android and iOS. That means the concepts and behaviors that apply to a regular native app also apply to an Expo app. If unsure about how a specific notification feature works on Android or iOS, see the [official documentation](#external-references) for the respective platform.
+
+## Local and Remote Notifications
+
+1. **Push Notifications**: (Also called "remote notifications") Notifications that are sent to a user's device from a remote server.
+2. **Local Notifications**: (Also called "in-app notifications") Notifications that are displayed from within the app.
```

Comment:
```suggestion
2. **Local Notifications**: (Also called "in-app notifications") Notifications that are created and displayed from within the app. Since many of the APIs that create these notifications will create them at a particular time, these may also sometimes be called "scheduled notifications". 
```